catalog/pages/bottom_sheet/index.md

```diff
@@ -14,7 +14,19 @@ rows:
   - Prop: withCancelBtn
     Type: boolean
     Default: true
-    Notes: Dictates whether to render the CancelButtonRow
+    Notes: Dictates whether to render the CancelButton
+  - Prop: withHeader
+    Type: boolean
+    Default: true
+    Notes: Dictates whether to render the HeaderRow
+  - Prop: label
+    Type: String
+    Default: ""
+    Notes: Optional
+  - Prop: labelVariant
+    Type: Number
+    Default: 0
+    Notes: Optional. Label styling. One of 0 (without styling), 1 (positive styling) or 2 (alert styling)
```

Comment:
Can you please name these variants semantically? Maybe `default`, `positive`, and `alert`?